main.go

```diff
@@ -132,77 +133,78 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 }
 
 // Get and return currently active milestones
-func getMilestones(baseURL string, token string, projectID string) (map[string]string, error) {
-	milestones := []milestoneAPI{}
-
-	strURL := []string{baseURL, projectID, "/milestones?state=active&per_page=100"}
-	URL := strings.Join(strURL, "")
-
-	m, linkHeader, err := callServer(URL, token, milestones)
-	if err != nil {
-		return m, err
-	}
-
-	for _, milestone := range milestones {
-		m[milestone.Title] = milestone.DueDate
-	}
-
-	parsedHeader := link.Parse(linkHeader)
-	for _, elem := range parsedHeader {
-		if elem.Rel == "next" {
-			m, _, err := callServer(URL, token, milestones)
-			if err != nil {
-				return m, err
-			}
-			for _, milestone := range milestones {
-				m[milestone.Title] = milestone.DueDate
-			}
-		}
-	}
+func getActiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "state=active"
+	return getMilestones(baseURL, token, projectID, state)
+}
 
-	return m, nil
+// Get and return inactive milestones
+func getInactiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "state=closed"
+    return getMilestones(baseURL, token, projectID, state)
 }
 
-func callServer(URL string, token string, milestones []milestoneAPI) (map[string]string, string, error) {
-	m := map[string]string{}
+func getMilestones(baseURL string, token string, projectID string, state string) ([]milestoneAPI, error) {
+	strURL := []string{baseURL, projectID, "/milestones?", state}
```

Comment:
The URL needs to be just the url. No manual additions of query strings. Examples on adding query values to a url: https://golang.org/src/net/url/example_test.go
This makes adding state as a query string easy.